app/models/manageiq/providers/nsxt/inventory/parser/network_manager.rb

```diff
@@ -48,81 +51,122 @@ def network_routers
 
   def cloud_networks
     collector.segments.each do |segment|
+      next if segment['type'] == 'DISCONNECTED'
```

Comment:
I'm guessing the refresher spec is failing due to this, if this is intentional then you just need to update the expected count